ddp-workspace/projects/ddp-sdk/src/lib/components/prism/prism.component.spec.ts

```diff
@@ -223,6 +216,14 @@ describe('PrismComponent', () => {
     expect(noUsersElement.textContent.trim()).toBe('NoUsers test');
   }));
 
+  it('sets initial data correctly', fakeAsync(() => {
```

Comment:
I believe you don't need `fakeAsync` here (and at line 403 as  well)

Reply:
fixed!